app/MicroserviceModule.scala

```diff
@@ -139,7 +141,12 @@ class MicroserviceModule(val environment: Environment, val configuration: Config
 }
 
 @Singleton
-class HttpVerbs @Inject() (val auditConnector: AuditConnector, @Named("appName") val appName: String)
-  extends HttpGet with HttpPost with HttpPut with HttpPatch with HttpDelete with WSHttp with HttpAuditing {
+class HttpVerbs @Inject()(
```

Comment:
i think you can make use of `DefaultHttpClient` from bootstrap that does all of this

Reply:
good point